src/screens/selectPlayers/index.tsx

```diff
@@ -83,16 +102,17 @@ const SelectPlayers = ({
       </Text>
       <FlatList
         bounces={false}
-        data={DATA_MOCK}
-        renderItem={renderItem}
+        data={players}
+        renderItem={({item}) => renderItem(item)}
```

Comment:
Update the renderItem typing and use it this way
```suggestion
        renderItem={renderItem}
```